doc/contribute/contributor_expectations.rst

```diff
@@ -213,17 +213,27 @@ defines the following escalation process for resolving technical disagreements.
 
   - Assignee to act as moderator if applicable.
 
-- Optionally resolve in the next `Zephyr Dev Meeting`_ or `Architecture Working
-  Group`_ meeting with more Maintainers and project stakeholders.
+- Resolve in the next `Zephyr Dev Meeting`_  meeting with more
+  Maintainers and project stakeholders.
 
-  - The involved parties and the Assignee to be present when
-    the (escalated) issue is discussed.
+  - The involved parties and the Assignee to be present when the  issue is
+    discussed.
 
-- TSC: Assignees can escalate to the TSC voting members and get a binding
-  resolution in the TSC by adding the `tsc`_ label on the PR.
+- If the above steps do not yield any results, the assignee (maintainer) has
+  the right to dismiss stale, unrelated or irrelevant change requests by
+  reviewers giving the reviewers a minimum of 1 business day to respond and
+  revisit their initial change requests or escalate to the `Architecture
+  Working Group`_.
```

Comment:
The minimum of 1 business day to respond is by responding in the PR I believe but not sure how to tie this with the "or" escalate to the arch WG? Should this be, 1 day to respond and if objecting, escalate to the next arch WG meeting?

Reply:
Yeah I also find the dynamics of this a bit awkward, how does merging plays into this? Review dismissed, should merging wait some time for the person to escalate?

Reply:
addressed....